main.go

```diff
@@ -39,35 +44,40 @@ func getOpenAPIHandler() http.Handler {
 func main() {
 	useTLS := strings.ToLower(os.Getenv("SERVE_HTTP")) != "true"
 
-	log := grpclog.NewLoggerV2(os.Stdout, ioutil.Discard, ioutil.Discard)
-	grpclog.SetLoggerV2(log)
+	logger := initLogger()
+
 	grpcport := os.Getenv("GRPC_PORT")
 	if grpcport == "" {
 		grpcport = "10000"
 	}
 	addr := "127.0.0.1:" + grpcport
 	lis, err := net.Listen("tcp", addr)
 	if err != nil {
-		log.Fatalf("failed to listen: %v", err)
+		logger.Fatal("failed to listen interface", zap.Error(err), zap.String("addr", addr))
 	}
 
 	var tlsConfig *tls.Config
 	if useTLS {
 		cert, err := tls.LoadX509KeyPair("certs/cert.pem", "certs/key.pem")
 		if err != nil {
-			log.Fatalf("failed to load key pair: %v", err)
+			logger.Fatal("failed to load key pair", zap.Error(err))
 		}
 
 		tlsConfig = &tls.Config{
 			Certificates: []tls.Certificate{cert},
 		}
 	}
-	s := grpc.NewServer()
 
+	s := grpc.NewServer(grpcServerLogOpt(logger))
 	pbVersion.RegisterVersionServiceServer(s, server.New())
-	log.Infof("serving gRPC on http://%s", addr)
+
+	logger.Info("serving gRPC", zap.String("Addr", "http://"+addr))
+
 	go func() {
-		log.Fatal(s.Serve(lis))
+		err := s.Serve(lis)
+		if err != nil {
+			logger.Fatal("Failed to serve grpc server", zap.Error(err))
+		}
```

Comment:
I think one line is ok here as it was.

Reply:
fixed fc4a972dc35840a06e1dd536ccfb1176527a749a